src/osgUtil/Optimizer.cpp

```diff
@@ -34,6 +34,8 @@
 #include <osg/Timer>
 #include <osg/TexMat>
 #include <osg/io_utils>
+//VRV_PATCH#
```

Comment:
We need to double check that any changes to this file are also made to DtOptimizer in VRV.